### PR TITLE
Tri-Force Cheat Sheet

### PR DESCRIPTION
**Pull request by chris0**
-----
IA Page: https://duck.co/ia/view/triforce_cheat_sheet

### CODE REVIEW

**Comment by chris0** [comment]
Updated

triforce

File: share/goodie/cheat_sheets/json/triforce.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+		},
+		{
+		  "key": "[Alt] [255]",
+		  "val": "Step 6"
+		},
+		{
+		  "key": "[Alt] [30]",
+		  "val": "Step 7"
+		}], 
+		"Mac": [{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/triforce.json]: Whoops, looks like your indentation is broken here..

File: share/goodie/cheat_sheets/json/triforce.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        {
+            "key": "[Option] [Space]",
+            "val": "Step 2"
+        },
+        {
+            "key": "[Option] [J]",
+            "val": "Step 3"
+        },
+        {
+           "key": "[Enter]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/triforce.json]: Please remove the `[]` around `Enter` it's only needed if you have multiple keys

File: share/goodie/cheat_sheets/json/triforce.json [hunk 3]
```diff
@@ -0,0 +1,103 @@
+{
+	"id": "triforce_cheat_sheet",
+	"name": "How To Tri-Force Instructional",
+	"description": "Step-by-step Tri-Forcing guide for Windows, Mac and Linux computers.",
+	"metadata": {
+		"sourceName": "eHow.com",
+		"sourceUrl": "http://www.ehow.com/how_8460794_triforce.html"
+	},
+	"aliases": ["triforce", "tri-force", "tri force"],
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/triforce.json]: You don't need a `triforce` alias as that's already the default trigger, based on the filename. The other 2 are fine.

File: share/goodie/cheat_sheets/json/triforce.json [hunk 4]
```diff
@@ -0,0 +1,103 @@
+{
+	"id": "triforce_cheat_sheet",
+	"name": "How To Tri-Force Instructional",
+	"description": "Step-by-step Tri-Forcing guide for Windows, Mac and Linux computers.",
+	"metadata": {
+		"sourceName": "eHow.com",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/triforce.json]: Let's remove the `.com` here

File: share/goodie/cheat_sheets/json/triforce.json [hunk 5]
```diff
@@ -0,0 +1,103 @@
+{
+	"id": "triforce_cheat_sheet",
+	"name": "How To Tri-Force Instructional",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/triforce.json]: Let's remove `Instructional`. "How to Tri-Force" or simply "Tri-Force" should be fine.

File: share/goodie/cheat_sheets/json/triforce.json [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+	},
+	"aliases": ["triforce", "tri-force", "tri force"],
+	"template_type": "keyboard",
+	"section_order": [
+		"Linux",
+		"Mac",
+		"Windows"
+	],
+	"sections": {
+		"Windows": [{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/triforce.json]: nit: Indentation is off a bit here.

File: share/goodie/cheat_sheets/json/triforce.json [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+        },
+        {
+            "key": "[Option] [Space]",
+            "val": "Step 6"
+        },
+        {
+            "key": "[Option] [J]",
+		   	"val": "Step 7"
+		}], 
+		"Linux": [{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/triforce.json]: nit: indentation

**Comment by moollaza** [comment]
@chris0 In the interest of getting this one merged, I'll quickly fix it up and merge it in :+1: 

Thanks again for contributing, we really appreciate it!

**Comment by moollaza** [comment]
@chris0 congrats! It's live: https://duckduckgo.com/?q=triforce+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by chris0** [comment]
@moollaza Thanks! I'll be sure to pick another one when I get a chance!

**Pull request merged**